spaceship/lib/spaceship/test_flight/build_trains.rb

```diff
@@ -13,12 +13,76 @@ class BuildTrains < Base
     # See `Spaceship::TestFlight::Build#reload`
 
     def self.all(app_id: nil, platform: nil, retry_count: 3)
-      data = client.get_build_trains(app_id: app_id, platform: platform)
-      trains = {}
+      client = Spaceship::ConnectAPI::Base.client
+
+      builds = []
+      included = []
+      cursor = nil
+
+      loop do 
+        builds_resp = client.get_builds(filter: { app: app_id, processingState: "VALID,PROCESSING,FAILED,INVALID"}, limit: 100, sort: "uploadedDate", includes: "buildBetaDetail,betaBuildMetrics,preReleaseVersion,app", cursor: cursor, only_data: false)
```

Comment:
I see here that `only_data` is set to `false`. Lets make this its own method on the client. Maybe
`client.get_builds_for_build_trains( ... )` or something like that?